addons/spectrum/functions/fnc_spectrumDeviceMouseDown.sqf

```diff
@@ -125,7 +125,8 @@ if (count _frequenciesSorted > 0) then {
 			_radio = true;
 			_timeActive = 0.1;
 		};
-		case "sweep";	// Players can not interact with a drone jammer. So just ignore this signal.
+		case "sweep_drone";	// Players can not interact with a drone jammer. So just ignore this signal.
+		case "sweep_radio";	// Players can not interact with a drone jammer. So just ignore this signal.
```

Comment:
Copy'n'pasted comment does not really fit `"sweep_radio"`.

Reply:
Good catch